sentinelone/plugin.spec.yaml

```diff
@@ -1580,6 +1580,74 @@ actions:
         description: Base64 encoded threat file
         type: file
         required: true
+  disable_agent:
+    title: Disable Agent
+    description: Disable agents that match the filter
+    input:
+      agent:
+        title: Agent
+        description: Agent to perform disable action on. Accepts IP address, MAC address, hostname, UUID or agent ID. Leave empty to perform action on all applicable Agents
+        type: string
+        example: hostname123
+        required: false
+      reboot:
+        title: Reboot
+        description: Set true to reboot the endpoint, set false to not reboot the endpoint
+        type: boolean
+        required: true
+        example: true
+      expiration_timezone:
+        title: Expiration Timezone
```

Comment:
We should do an enumerated list of timezones to make it easier for the user to choose the right string

Reply:
Does the enable action not support time zones? notice it didn't have this input

Reply:
Hi @jonschipp  :) Enable doesn't support time zones so it's only filter and reboot input.
We are waiting fore response from SentinelOne about supported timezones :)

Reply:
@r7-kszczepanskagorna e-mail sent with details